pyproject.toml

```
@@ -31,6 +31,8 @@ pytest = "^7.1.2"
 gitpython = "^3.1.27"
 toml = "^=0.10.2"
 
+[tool.poetry.plugins."console_scripts"]
```

```suggestion
[tool.poetry.scripts]
```
Is the appropriate section you want to use, [see](https://python-poetry.org/docs/pyproject/#scripts)

---

scripts gives me different behavior with installation via poetry install vs. installation using generated setup.py. So i will stick with console_scripts for consistency.